modules/file-integrity-important-attributes.adoc

```diff
@@ -32,11 +32,15 @@ a default toleration is applied, which allows tolerations to run on control plan
 checks on a node can be resource intensive, so it can be useful to specify a
 longer interval. Defaults to `900`, or 15 minutes.
 
-|`spec.config.name`, `spec.config.namespace`, `spec.config.key`
-|These three attributes allow you to set a custom AIDE configuration. When the name
-or namespace are unset, the File Integrity Operator generates a configuration
-suitable for {op-system} systems. The name and namespace attributes point to the
-config map; the key points to a key inside that config map. Use the key
-attribute to specify a custom key that contains the actual config and defaults
-to `aide.conf`.
+|`maxBackups`
+|The maximum number of AIDE database and log backups (leftover from the re-init process) to keep on a node. Older backups beyond this number are automatically pruned by the daemon.
+
+|`spec.config.name`
+| Name of a configMap that contains custom AIDE configuration. A default configuration is created if omitted.
+
+|`spec.config.namespace`
+|Namespace of a configMap that contains custom AIDE configuration. A default configuration is created if omitted.
```

Comment:
A default configuration is used if the _namespace_ is omitted?

Or does it use some default namespace to look for the config map object if not specified? Looks like the answer was here:

> When the name
> or namespace are unset, the File Integrity Operator generates a configuration
> suitable for {op-system} systems.